src/index.html

```diff
@@ -17,6 +21,86 @@
     />
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header>
+      <div class="logo-container">
+        <a href="index.html">
+          <img
+            src="./images/logo.png"
+            alt="MOYO Logo"
+            class="logo"
+          />
+        </a>
```

Comment:
Don't use spaces in `<a>` tag's `href` property for internal navigation. Use `#` followed by the anchor name or leave it empty if it's a placeholder.